requirements/production.txt

```diff
@@ -4,6 +4,7 @@ django-allauth==0.38.0
 django-bootstrap4
 django-celery-beat==1.1.*
 django-js-reverse==0.8.2
+django-massmailer
```

Comment:
Can you pin a version?

Reply:
Considering I'm still unbreaking stuff on it, I don't really advise it. You're the first early adopters!